src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -65,6 +65,9 @@ private enum ResponseStatus
             long errorCode = authResult.Error.ErrorCode;
             string errorMessage;
 
+            logger.Info("[WamBroker] Processing WAM exception");
+            logger.Verbose($"[WamBroker] TelemetryData: {authResult.TelemetryData}");
```

Comment:
nit: Any benefit to readability of putting this after the switch statement, so that actual error is printed first?

Reply:
after or inside each case? we throw when there is an error so the code won't be reachable

Reply:
Ah, yep, nevermind then.